presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestAddExchangesPlans.java

```diff
@@ -422,9 +422,7 @@ void assertExactDistributedPlan(String sql, PlanMatchPattern pattern)
                 TestingSession.testSessionBuilder()
                         .setCatalog("local")
                         .setSchema("tiny")
-                        .setSystemProperty(
-                                SystemSessionProperties.PARTITIONING_PRECISION_STRATEGY,
-                                FeaturesConfig.PartitioningPrecisionStrategy.PREFER_EXACT_PARTITIONING.toString())
+                        .setSystemProperty(PARTITIONING_PRECISION_STRATEGY, PREFER_EXACT_PARTITIONING.toString())
```

Comment:
Curious why it's called `PARTITIONING_PRECISION_STRATEGY`? -- didn't see discussion in https://github.com/prestodb/presto/pull/13354 ? 

Since when I see "precision", I am thinking whether the "partitioning" is accurate or not. However, in this case it's about whether we want to use **ALL** the columns required by join/aggregation, while any partitioning with *SUBSET** of the columns would work.

Reply:
We have a limited set of columns we can choose to partition on and the order matters. It must be partitioned on a prefix of the columns in the order used in the join/aggregation.

So we have different options for choosing whether to prefer precisely partitioning on all the columns (until the Presto learns to do a better job), or allowing Presto to choose (automatic).

Precision doesn't mean inaccurate especially in math and engineering. It just means accurate up to a point. The partitioning always needs to be accurate we just have a choice as to how big a prefix of columns to choose.

I don't really care what it's called, but it made sense to me at the time.